src/SFA.DAS.ApplyService.Application/Apply/Submit/SubmitApplicationHandler.cs

```diff
@@ -29,10 +29,8 @@ public async Task<bool> Handle(SubmitApplicationRequest request, CancellationTok
 
                 if (application.ApplyData != null && submittingContact != null)
                 {
-                    if (application.ApplyData.ApplyDetails == null)
-                    {
-                        application.ApplyData.ApplyDetails = new ApplyDetails();
-                    }
+                    application.ApplyData.ApplyDetails = request.ApplyData.ApplyDetails;
```

Comment:
Null Reference if ApplyDetails is null for some reason. I know it shouldn't be though...